torchvision/models/regnet.py

```diff
@@ -864,8 +864,16 @@ def regnet_y_400mf(*, weights: Optional[RegNet_Y_400MF_Weights] = None, progress
     `"Designing Network Design Spaces" <https://arxiv.org/abs/2003.13678>`_.
 
     Args:
-        weights (RegNet_Y_400MF_Weights, optional): The pretrained weights for the model
-        progress (bool): If True, displays a progress bar of the download to stderr
+        weights (:class:`torchvision.models.regnet.RegNet_Y_400MF_Weights`, optional): The pretrained weights to use. See
+        :class:`~torchvision.models.regnet.RegNet_Y_400MF_Weights` below for more details and possible values. By default,
+        no pretrained weights are used.
+        progress (bool, optional): If True, displays a progress bar of the download to stderr. Default is True.
+        **kwargs: parameters passed to either ``torchvision.models.regnet.RegNet`` or
+        ``torchvision.models.regnet.BlockParams`` class. Please refer to the `source code
+        <https://github.com/pytorch/vision/blob/main/torchvision/models/regnet.py>`_ for more detail about the classes.
```

Comment:
Here and in the other docstrings, there is an indentiation issue that causes the docs to not render properly: see e.g. https://output.circle-artifacts.com/output/job/e5544cc3-176f-47df-ac8c-c2674abe5489/artifacts/0/docs/models/generated/torchvision.models.regnet_y_400mf.html#torchvision.models.regnet_y_400mf

Reply:
ah got it, will fix all these indentation issue!